web/models/check.go

```diff
@@ -28,7 +28,10 @@ type Results struct {
 
 // The ChecksByHost struct stores the checks list, but the results are grouped by hosts
 type ChecksByHost struct {
-	Hosts map[string]*Check `json:"hosts,omitempty" mapstructure:"hosts,omitempty"`
+	ID          string            `json:"id,omitempty" mapstructure:"id,omitempty"`
```

Comment:
I don't really get the purpose of this change. Is this to send the information in the new API handler?
Maybe we should create new struct.
This was originally designed to parse the data coming from ARA.
Anyway, we can discuss it

Reply:
I think you're actually right, let's see if I can come up with something better